exe/kubernetes-render

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/render_task'
+require 'optparse'
+
+template_dir = nil
+bindings = {}
+
+ARGV.options do |opts|
+  opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates " \
+    "(format: k1=v1,k2=v2, JSON string or file (JSON or YAML) path prefixed by '@')") do |binds|
+    bindings.merge!(KubernetesDeploy::BindingsParser.parse(binds))
+  end
+  opts.on("--template-dir=DIR", "Set the template dir (default: config/deploy/$ENVIRONMENT)") { |v| template_dir = v }
+  opts.parse!
+end
+
+template = ARGV[0]
+logger = KubernetesDeploy::FormattedLogger.build('test', 'test', verbose_prefix: false)
+revision = KubernetesDeploy::OptionsHelper.revision_from_environment
+
+if !template || template.empty?
+  puts "A template must be provided as an argument to the program."
```

Comment:
I was thinking about this too and I think that a lot of folks will indeed have that use case of rendering a lot of stuff in a directory, but, I think the main use cases for this new program are 
generally oriented towards rendering one file, not a whole directory. The use cases in my mind are rendering in a CI pipeline to validate, and rendering in development to understand how the heck the thing works. For my CI pipeline example above, I actually don't want to render the whole group at once so that if rendering one file fails, or kubeval fails, I can output a rich error message to the user about which file it actually was that caused the error. That's all outer-program logic that composes kubernetes-render of course but I think that kubernetes-render would be best used in the same way.

That said, it'd be convenient to render thole whole template dir sometimes I imagine if you don't care about file-wise error messages from downstream tooling, so I could add it. `cat` supports catting multiple files, I don't see why I should be so much more persnickety. 

Do we do that `template-dir` is the parent assumption anywhere else? I feel like we could definitely do what you described with the mutually exclusive options, or adopt a `cat` style interface where all of `ARGV` after options parsing is treated as templates that we loop over and render. That puts the onus of globbing on the user but is a bit more unix-y an interface.

Reply:
I discussed this with the team today, and we're thinking it would make sense to support using `--template-dir=/path/to/dir` on its own to render all the files in the directory, or `--template-dir=/path/to/dir FILENAME` to render a specific file only (that file must be in the template dir, per your current implementation). We also agreed that your `cat` idea to support rendering multiple specific files in the directory would be great, but we wouldn't consider it a requirement for this PR. Does that sound reasonable to you?

Reply:
Ok, updated to do this.